### DISCUSSION TITLE
3차 데모데이 피드백

### DISCUSSION THREAD

**Discussion by Hain-tain** [woowacourse-teams/2024-code-zap#387] [QA]
3차 데모데이 피드백

**Comment by zangsu** [comment]
화면이 훨씬 좋아졌다 -> 이제는 우리 서비스가 어떤 서비스인지 알 것 같다.
위키와 디스커션을 서로 함께 링크로 관리하면 좋을 것 같다!

**Comment by Hain-tain** [comment]
에러로깅 관련 논의 추가로 필요하다!
- 알림을 받으면 끝나는게 아니라 처리를 하기 위해 받는것!!
- 다음 프로세스를 어떻게 할 지 고민해야 한다.
- 버그픽스 이슈가 생기면 어떻게 반영할건지, 어떤 흐름으로 반영을 할건지 논의해야 한다.
- 모니터링 알림이 오면=> 최종 반영까지 어떻게 할건지 얘기해보면 좋을 것 같다.
- 프로세스가 잘 돌아가려면 어떤 에러를 받아야 하는지도 운영/배포 나가기 전에 다시 가다듬으면 좋을 것 같다.

**Comment by HoeSeong123** [comment]
- 이제 뭘 하는 지 대충 알 것 같다.
- 왜 그렇게 정했는지 논의한 내용들이 링크나 문서로 잇었으면 좋겠다.
- 모니터링 템플릿이 다들 비슷하네요...ㅋㅋ;;
- 버그 픽스 주기? 등을 어떤 식으로 가져갈지, 어떤 흐름으로 반영할지 등등
- 에러가 났을 때 사용자에게 피드백이 안 온다.
- 홈화면이 처음에 나왔으면 좋겠다.
- 코드잽의 타겟에 대한 설명, 왜 그런 분에게 우리 서비스가 필요한가??

**Comment by zangsu** [comment]
회원가입이 안됐을 때 피드백이 없다.
랜딩 페이지를 처음에 더 잘 보여주자
-> 우리 서비스가 왜 필요한지를 설득하는 내용이 들어가면 더 좋겠다.

**Comment by Hain-tain** [comment]
<코딩해듀오팀 피드백> 이지만 우리도 하면 좋을 것!

- 의사결정을 할 땐, 문서화해라! 우리 팀에 누군가 합류했을 때 근거로 보여줄 수 있도록!

**Comment by Jaymyong66** [comment]
### 개인적인 피드백
- 회원가입 페이지가 우리 서비스의 사용자가 처음 이용하는 기능인데, 생각보다 불친절했다. UX를 친절하게, 그리팅을 잘해주자.
